netket/utils/__init__.py

```diff
@@ -31,6 +31,8 @@
 
 from .history import History, accum_in_tree, accum_histories_in_tree
 
+from .float_utils import comparable, comparable_periodic, prune_zeros, is_approx_int
```

Comment:
Also, I think you should import symmetry explicitly?

Reply:
No, because folders (not files) are allowed by `hide_submodules`

Reply:
> can we just import the module and not the stuff inside?

But then let's not name it `utils.float_utils` please. :slightly_smiling_face:

Reply:
No, because folders (not files) are allowed by hide_submodules. You only have to do
```
from . import types
```
for instance because hide_submodules removes them. But it doesn't remove folders. (Didn't you write that module?)

Reply:
`hide_submodules` only hides them (renames them from `name` to `_name`), they are still loaded.
it's simply a matter of loading order.
Nothing else.
But if it becomes its own thing this is not important

Reply:
Yeah, it seems it doesn't mind folders being around. Also, it's so good at removing files that even once you write `from . import types` in `__init__`, it's still not available from the console